mantle/platform/qemu.go

```diff
@@ -555,7 +586,8 @@ func (builder *QemuBuilder) addDiskImpl(disk *Disk, primary bool) error {
 	if disk.SectorSize != 0 {
 		diskOpts = append(diskOpts, fmt.Sprintf("physical_block_size=%[1]d,logical_block_size=%[1]d", disk.SectorSize))
 	}
-	builder.addQcow2DiskFd(fd, channel, diskOpts)
+
+	builder.addQemuDiskFd(fd, channel, disk, diskOpts)
```

Comment:
`Qemu` is redundant here right?  Might as well just be `addDiskFd`.

Reply:
Indeed. Dropped.